src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_NUMBER = 5;
+    public static final int DIMENSIONS_LIMITER = 50;
+    private final Random random = new Random();
+    private final ColorSupplier colors = new ColorSupplier();
+    //private final String figureColor = colors;
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(FIGURE_NUMBER);
+        int randomDimension1 = random.nextInt(DIMENSIONS_LIMITER) + 1;
+        int randomDimension2 = random.nextInt(DIMENSIONS_LIMITER) + 1;
+        int randomDimension3 = random.nextInt(DIMENSIONS_LIMITER) + 1;
```

Comment:
think how to get rid of repeating code lines cause not all figures need randomDimension3 for example